Symfony/CS/ConfigurationResolver.php

```diff
@@ -84,6 +99,10 @@ public function getFixers()
 
     public function getProgress()
     {
+        if (!method_exists($this->config, 'getHideProgress')) {
```

Comment:
Oh, you mean `Config`, not `ConfigInterface`.
That could work too, but I think calling `method_exists` is more generic